File: IT/2/Programmazione funzionale/Esercizi/Java/Esercizio4.java
/* file Esercizio4.java */

import java.util.Collections;
import java.util.List;
import java.util.Random;
import java.util.stream.Collectors;

public class Esercizio4 {
	public static void main(String [] args) {
		List<Integer> li = new Random().ints(10, 1, 100).boxed().collect(Collectors.toList());
		System.out.println(li);

		List<Integer> l2 = li.stream().sorted().collect(Collectors.toList());
		System.out.println(l2);

		List<Integer> l3 = li.stream().sorted(Collections.reverseOrder()).collect(Collectors.toList());
		System.out.println(l3);
	}
}

File: IT/2/Programmazione funzionale/Esercizi/Python/esercizio4.py
# file esercizio4.py

from __future__ import print_function
from random import randint

li = [randint(1, 100) for n in range(10)]

l2 = sorted(li)
l3 = sorted(li, reverse=True)

print(li)
print(l2)
print(l3)

